components/distro-packages/python-Cython/SPECS/python-Cython.spec

```diff
@@ -21,7 +21,8 @@ License:        Apache-2.0
 Group:          %{PROJ_NAME}/distro-packages
 Source0:        https://files.pythonhosted.org/packages/source/C/Cython/Cython-%{version}.tar.gz
 Source1:        python-Cython-rpmlintrc
-%if 0%{?rhel}
+%if 0%{?rhel} || 0%{?openEuler}
+BuildRequires: libstdc++-devel
```

Comment:
https://github.com/Yikun/ohpc/actions/runs/3928230507/jobs/6715621105
 
This line fix the error in RHEL:
```
2023-01-16 07:46:39,892 ERROR
./components/distro-packages/python-Cython/SPECS/python-Cython.spec
Source42: OHPC_macros
Source0:        https://files.pythonhosted.org/packages/source/C/Cython/Cython-0.29.32.tar.gz
Trying to get https://files.pythonhosted.org/packages/source/C/Cython/Cython-0.29.32.tar.gz
Source1:        python-Cython-rpmlintrc
Building SRPM for components/distro-packages/python-Cython/SPECS/python-Cython.spec
/github/home/rpmbuild/SRPMS/python3-Cython-ohpc-0.29.32-1.el8.src.rpm
Last metadata expiration check: 0:00:11 ago on Mon Jan 16 07:46:28 20[23](https://github.com/Yikun/ohpc/actions/runs/3928230507/jobs/6715621105#step:7:24).
Package python3-rpm-macros-3-43.el8.noarch is already installed.
Package python3-setuptools-39.2.0-6.el8.noarch is already installed.
Error: 
 Problem: cannot install the best candidate for the job
  - nothing provides libstdc++ = 8.5.0-16.el8_7 needed by gcc-c++-8.5.0-16.el8_7.x86_64
```